### PR TITLE
Features/docu 506

### PR DESCRIPTION
**Pull request by benradstone**

### CODE REVIEW

File: docs/quickstarts/testing/test_hdp_sandbox.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+id: test_hdp_sandbox
+title: HDP Sandbox
+sidebar_label: HDP Sandbox
```
Comment (catrionabarker) [inline docs/quickstarts/testing/test_hdp_sandbox.md:4]: I think title and sidebar need to include test

File: docs/quickstarts/testing/test_hdp_sandbox.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+_Example_
+
+To generate 10GB of data inside a replicated path, run:
+
+`docker-compose exec -u hdfs sandbox-hdp hadoop jar /usr/hdp/2.6.5.0-292/hadoop-mapreduce/hadoop-mapreduce-examples.jar teragen 100000000 /path/to/replication_rule/teragen_output`
+
+The `teragen_output` directory should not be created prior to running this command otherwise it will fail.
+
+Once complete, check the storage on your target zone for the generated files in the `teragen_output` directory. You will see a `_SUCCESS` file alongside the generated files once it is complete.
```
Comment (catrionabarker) [inline docs/quickstarts/testing/test_hdp_sandbox.md:41]: 'once it is complete' - can we be more specific than 'it'

File: docs/quickstarts/testing/test_hdp_sandbox.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+### TeraSort
+
+Use the `terasort` option to sort (i.e. organize) the generated data in to a replicated path:
```
Comment (catrionabarker) [inline docs/quickstarts/testing/test_hdp_sandbox.md:45]: into

File: docs/quickstarts/testing/test_hdp_sandbox.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+`docker-compose exec -u hdfs sandbox-hdp hadoop jar /usr/hdp/2.6.5.0-292/hadoop-mapreduce/hadoop-mapreduce-examples.jar terasort <input-path> <output-path>`
+
+_Example_
+
+To sort the data from the staging directory in to a replicated path, run:
```
Comment (catrionabarker) [inline docs/quickstarts/testing/test_hdp_sandbox.md:51]: into

File: docs/quickstarts/testing/test_hdp_sandbox.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+Use the `teravalidate` option to test that the data in the replicated path is now globally sorted:
+
+`docker-compose exec -u hdfs sandbox-hdp hadoop jar /usr/hdp/2.6.5.0-292/hadoop-mapreduce/hadoop-mapreduce-examples.jar teravalidate <input-path> <output-path>`
+
+_Example_
+
+`docker-compose exec -u hdfs sandbox-hdp hadoop jar /usr/hdp/2.6.5.0-292/hadoop-mapreduce/hadoop-mapreduce-examples.jar teravalidate /path/to/replication_rule/terasort_output /path/to/replication_rule/teravalidate-output`
+
+If everything is correctly sorted, the `teravalidate-output` directory should contain a `_SUCCESS` file and another file containing a checksum value. Compare this value on both the source and target storages if you wish to fully validate the operation.
```
Comment (catrionabarker) [inline docs/quickstarts/testing/test_hdp_sandbox.md:69]: To fully validate the operation, you can compare this value on both the source and target storages.

That reads better to me but it's trivial!

File: docs/quickstarts/testing/test_hdp_sandbox.md (hunk 2 above)
Comment (benradstone) [reply docs/quickstarts/testing/test_hdp_sandbox.md]: I've just gone and removed this. It still makes sense to me now.

**Pull request merged**